languages/rust/oso/src/query.rs

```diff
@@ -265,18 +265,16 @@ impl ResultSet {
         self.bindings.is_empty()
     }
 
-    pub fn get(&self, name: &str) -> Option<crate::Value> {
+    pub fn get(&self, name: &str) -> Option<crate::PolarValue> {
```

Comment:
If this only returns PolarValue now, how would an external instance be returned?